tests/FileStatus.hs

```diff
@@ -70,24 +71,16 @@ testLink = do
   createLink regular hlink_regular
   (fs, _)  <- getStatus regular -- we need to retrieve it again as creating the link causes it to change!
   (fs', ls)  <- getStatus hlink_regular
-  let expected = (
+  snd (statusElements ls) @?= (
                 False, -- isBlockDevice
                 False, -- isCharacterDevice
                 False, -- isNamedPipe
                 True,  -- isRegularFile
                 False, -- isDirectory
                 False, -- isSymbolicLink
                 False) -- isSocket
-      actualF  = snd (statusElements ls)
-
-  when (actualF /= expected) $
-    fail "unexpected file status bits for hard link to regular file"
-
-  when (linkCount fs' /= 2) $
-    fail "newly created hard link was expected to contain have a link count of 2"
-
-  when (statusElements fs /= statusElements fs') $
-    fail "status for a file does not match when it's accessed via a link"
+  linkCount fs' == 2 @? "Newly created hard link was expected to contain have a link count of 2"
```

Comment:
It's probably either "to contain" or "to have"?..

Reply:
good catch, yes. will fix in next push